yb-voyager/src/migassessment/assessmentDB.go

```diff
@@ -0,0 +1,229 @@
+/*
+Copyright (c) YugabyteDB, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package migassessment
+
+import (
+	"database/sql"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/metadb"
+	"github.com/yugabyte/yb-voyager/yb-voyager/src/utils"
+)
+
+const (
+	TABLE_INDEX_IOPS           = "table_index_iops"
+	TABLE_INDEX_SIZES          = "table_index_sizes"
+	TABLE_ROW_COUNTS           = "table_row_counts"
+	COLUMNS_COUNT              = "columns_count"
+	INDEX_TO_TABLE_MAPPING     = "index_to_table_mapping"
+	TABLE_COLUMNS_DATA_TYPES   = "table_columns_data_types"
+	MIGRATION_ASSESSMENT_STATS = "migration_assessment_stats"
+)
+
+func GetDBFilePath() string {
+	return filepath.Join(AssessmentDataDir, "assessment.db")
+}
+
+func CreateAndInitAssessmentDB() error {
+	assessmentDBPath := GetDBFilePath()
+	err := createAssessmentDBFile(assessmentDBPath)
+	if err != nil {
+		return err
+	}
+
+	err = initAssessmentDB(assessmentDBPath)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func createAssessmentDBFile(assessmentDBPath string) error {
+	log.Infof("creating assessment db file at %s", assessmentDBPath)
+	file, err := os.Create(assessmentDBPath)
+	if err != nil {
+		return fmt.Errorf("error creating assessment db file %s: %w", assessmentDBPath, err)
+	}
+
+	err = file.Close()
+	if err != nil {
+		return fmt.Errorf("error closing assessment db file %s: %w", assessmentDBPath, err)
+	}
+	return nil
+}
+
+func initAssessmentDB(assessmentDBPath string) error {
+	log.Infof("initializing assessment db at %s", assessmentDBPath)
+	conn, err := sql.Open("sqlite3", fmt.Sprintf("%s%s", assessmentDBPath, metadb.SQLITE_OPTIONS))
```

Comment:
Doesn't this create a new file if it doesn't exist? If it does, do we need `createAssessmentDBFile()`?